File: OPENAPI-REST-API/openapi-client/scala-gatling/src/gatling/scala/org/openapitools/client/model/PostOauthCallbackRequest.scala

package org.openapitools.client.model


case class PostOauthCallbackRequest (
    /* The OAuth provider name (e.g. `Google`). */
    _provider: Option[String]
)
object PostOauthCallbackRequest {
    def toStringBody(var_provider: Object) =
        s"""
        | {
        | "provider":$var_provider
        | }
        """.stripMargin
}

File: OPENAPI-REST-API/openapi-client/php-dt/src/App/DTO/PostOauthCallbackRequest.php
<?php
declare(strict_types=1);

namespace App\DTO;

use Articus\DataTransfer\PhpAttribute as DTA;

class PostOauthCallbackRequest
{
    /**
     * The OAuth provider name (e.g. &#x60;Google&#x60;).
     */
    #[DTA\Data(field: "provider", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $provider = null;

}
